### ISSUE TITLE
Choose multiple polls

### ISSUE THREAD

**Issue by DInnoStudio** [adamosimpson21/wepollclient#10]
Priority: High

The ability to create a poll where a user can choose multiple answers (0-all).  Creates a histogram chart.

**Comment by adamosimpson21** [comment]
Difficulty: 4
If I can, I'd like to group this together with other possible answer types. Are there any you can think of that would be useful right now? A range type, where a user could potentially have a min and max for a slider? Those kinds of questions could be represented by histograms as well. Imagine "best time of day to do X" or "best ages to get married" or "a 35 yo male can get married to an X year old person" for example. Obivously not perfect questions, but that's a possibility, and if I could build all of those functionalities at once, it'd be much easier than patching the same thing 4-5 times

**Comment by DInnoStudio** [comment]
Yes I'd certainly like a slider, like you described.  With the question creator able to set the min and max to read whatever they want.  How many intervals can you make it have?  If someone wanted to say 0-24 hrs, could they have 25 intervals?  Can we do 100?  If we wanted to do something like a range of freedom - security, for instance, could we let people select 83%/17% ?
Another type of question answer I would like is a "Enter a number" with a blank box that accepts integers up to like 10 trillion.  So someone can ask for instance "What should the max federal deficit be?" or "How much oil should the US buy from Saudi Arabia".  (If the slider would be better off with 10 or fewer intervals, we could use this Answer type for 11 or larger answers)

**Comment by adamosimpson21** [comment]
The slider is very customizable. It takes a little bit of time to build all of these pieces, but they're all feasible. The main concern I'd have is data sparsity. Imagine we have a range from 0-9999999999 etc. , how do you extract meaningful data from that? Even if 1000 people answer the question, what are the chances 2 people will have the same answer. I think for large numbers (over 10 or so) it'd be best to use ranges. ex: <1 billion, 1-2 bil, 3-5 bil, 6+ bil. You can also have a range with steps, 0, 1bil, 2bil, 3bil, etc. or ask the question in a more precise way, how many billions of dollars of debt, etc. Something like a percent can be done in steps pretty easily: 0%, 10%, 20%, etc. We probably want to avoid more than 10 or so options for any question due to data sparsity. People round percentages, dollar amounts, years, all the time anyway xD

**Comment by DInnoStudio** [comment]
Bump.  Need this as a feature to get people from twitter to website.
